### ISSUE TITLE
Send more than one image

### ISSUE THREAD

**Issue by trufae** [Antidote-for-Tox/Antidote#272]
Being able to select more than one image from the camera roll to send to a contact.

**Comment by dvor** [comment]
It would be nice to have some custom image picker with multiple selection, like most of other messengers now do.

**Comment by trufae** [comment]
agree, by googling a bit i’ve found a bunch of libraries that do that, here’s the list, 9 of them in Swift: http://stackoverflow.com/questions/20756899/how-to-select-multiple-images-from-uiimagepickercontroller <http://stackoverflow.com/questions/20756899/how-to-select-multiple-images-from-uiimagepickercontroller>

This is anounced as a whatsapp style image picker.. but would be good to compare with others before choosing it

https://github.com/limhowe/LimPicker <https://github.com/limhowe/LimPicker>
